pos_longpolling/static/src/js/pos_longpolling.js

```diff
@@ -45,8 +45,13 @@ odoo.define('pos_longpolling', function(require){
                 if(!self.stop){
                     self.poll();
                 }
-
+                //new condition: change longpolling icon status if needed
+                if (!self.longpolling_connection.status) {
+                    self.longpolling_connection.network_is_on();
+                }
             }, function(unused, e) {
+                //new line: change longpolling icon status
```

Comment:
When developer read the code first time he doesn't understand what is "new". All lines were new at some moment